plugins/SkyCultureMaker/src/types/SkyPoint.hpp

```diff
@@ -34,10 +34,17 @@ namespace scm
 struct SkyPoint
 {
 	//! The coordinate of the point.
-	Vec3d coordinate;
+	Vec3d coordinate = Vec3d(0, 0, 0);
 
 	//! The optional star at that coordinate.
-	std::optional<QString> star;
+	std::optional<QString> star = std::nullopt;
```

Comment:
Why does it have to be an optional? Isn't `QString::isEmpty()` sufficient for checking that a string is present?